lib/vis_milk2/ns-eel2/scripts/circle.eel

```diff
@@ -0,0 +1,50 @@
+gfx_init("vis",1024,768);
```

Comment:
I thinks `scripts` folder also can be dropped.

Reply:
yep